src/libs/OptionsListUtils.ts

```diff
@@ -1,13 +1,21 @@
 /* eslint-disable no-continue */
 import {parsePhoneNumber} from 'awesome-phonenumber';
 import Str from 'expensify-common/lib/str';
+// eslint-disable-next-line you-dont-need-lodash-underscore/get
 import lodashGet from 'lodash/get';
```

Comment:
We should remove usage of `lodashGet`

Reply:
I think in that case its good to use `lodashGet`. I am using this in this case 
`const existedValue = lodashGet(hierarchy, path, {});` where path is string[] I tried to write a native equivalent but its really hard to write it in typescript :/